app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -239,4 +253,27 @@ private void importDatabase(final String filePath) {
             ErrorActivity.reportUiErrorInSnackbar(this, "Importing database", e);
         }
     }
+
+    private boolean isValidPath(final String path) {
+        if (path == null || path.isEmpty()) {
+            return false;
+        }
+        final File file = new File(path);
+        return file.exists() && file.isDirectory();
+    }
```

Comment:
Can this be moved out of the fragment into a Helper class and unit tests added for it.

Reply:
Good idea, I'll fix it when I find free time. thanks.

Reply:
fixed